modules/cookbooks/pages/kafka_migrator.adoc

```diff
@@ -0,0 +1,352 @@
+      KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP: CONTROLLER:PLAINTEXT,PLAINTEXT:PLAINTEXT,PLAINTEXT_DOCKER:PLAINTEXT
+      KAFKA_CFG_LISTENERS: PLAINTEXT://0.0.0.0:9092,PLAINTEXT_DOCKER://0.0.0.0:19092,CONTROLLER://:9093
+      KAFKA_CFG_ADVERTISED_LISTENERS: PLAINTEXT://localhost:9092,PLAINTEXT_DOCKER://source:19092
+      KAFKA_CFG_AUTHORIZER_CLASS_NAME: "org.apache.kafka.metadata.authorizer.StandardAuthorizer"
+      KAFKA_CFG_SUPER_USERS: User:redpanda;User:ANONYMOUS
+    ports:
+      - 9092:9092
+      - 19092:19092
+    healthcheck:
+      test: [ "CMD", "kafka-topics.sh", "--bootstrap-server=localhost:9092", "--list" ]
+      start_period: 5s
+      interval: 3s
+
+  init_source:
+    image: bitnami/kafka
+    working_dir: /opt/bitnami/kafka/bin
+    entrypoint: /bin/bash
+    depends_on:
+      source:
+        condition: service_healthy
+    command: >
+      -c "kafka-topics.sh --create --if-not-exists --topic foo --replication-factor=1 --partitions=2 --bootstrap-server source:19092 &&
+          kafka-topics.sh --create --if-not-exists --topic bar --replication-factor=1 --partitions=2 --bootstrap-server source:19092 &&
+          echo 'Created topics:' &&
+          kafka-topics.sh --list --exclude-internal --bootstrap-server source:19092 &&
+          kafka-acls.sh --bootstrap-server source:19092 --add --allow-principal User:redpanda --operation Read --topic foo &&
+          kafka-acls.sh --bootstrap-server source:19092 --add --deny-principal User:redpanda --operation Read --topic bar
+          echo 'Created ACLs:' &&
+          kafka-acls.sh --bootstrap-server source:19092 --list"
+
+  source_schema_registry:
+    image: bitnami/schema-registry
+    environment:
+      SCHEMA_REGISTRY_KAFKA_BROKERS: PLAINTEXT://source:19092
+    ports:
+      - 8081:8081
+    depends_on:
+      source:
+        condition: service_healthy
+
+  destination:
+    image: redpandadata/redpanda
+    command:
+      - redpanda
+      - start
+      - --node-id 0
+      - --mode dev-container
+      - --set rpk.additional_start_flags=[--reactor-backend=epoll]
+      - --set redpanda.auto_create_topics_enabled=false
+      - --kafka-addr 0.0.0.0:9093
+      - --advertise-kafka-addr localhost:9093
+      - --schema-registry-addr 0.0.0.0:8081
+    ports:
+      - 8082:8081
+      - 9093:9093
+      - 9645:9644
+----
+
+[source,console]
+----
+$ docker-compose -f docker-compose.yml up --force-recreate -V
+----
+
+NOTE: You used an `init` container above to create 2 topics, `foo` and `bar`, each with 2 partitions and an associated ACL.
+
+== Create schemas
+
+Once the demo clusters are up and running, use https://curl.se[curl^] to create a schema for each topic in the `source` cluster.
+
+[source,console]
+----
+$ curl -X POST -H "Content-Type: application/vnd.schemaregistry.v1+json" --data '{"schema": "{\"name\": \"Foo\", \"type\": \"record\", \"fields\": [{\"name\": \"data\", \"type\": \"int\"}]}"}' http://localhost:8081/subjects/foo/versions
```

Comment:
Please exclude `$` from bash commands. They get copied into the user's terminal and cause the examples to break.